packages/manager/apps/pci-instances/src/queryClient.ts

```diff
@@ -4,6 +4,7 @@ const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
       staleTime: 300_000,
+      throwOnError: true,
```

Comment:
This will cause the ErrorBoundary to be mounted each time an error is thrown, and we don't want this behavior as we use error banners to be displayed in specific use cases instead of the fallback UI

Reply:
Since there is no page with ErrorBanner and the ErrorBanner takes the whole page I don't understand how you would handle the error ? If this setting is set to false there is no error thrown and the error banner isn't displayed